### ISSUE TITLE
More URL broken

### ISSUE THREAD

**Issue by ljwolford** [adlnet/xapi-launch#7]
I registered an online app for a local version of https://github.com/creighton/itsec2014charts
Whenever I try to use the Live LRS data in coursechart.html, the stmtdash calls fetchAllStatements and attempts to grab all statements including those from more links ...

```
		wrapper.getStatements(query, null, function getMore(r){
			var response = JSON.parse(r.response);
			
			self.data.append(response.statements);

			if(response.more){
				wrapper.getStatements(null, response.more, getMore);
			}
			else {
				cb(self.data);
			}
```

The launch server is adding 'null' to the end of the more link so it is returning an empty object, and this function is appending undefined to its own data because of it. Example of URL launch is attempting:

`https://lrs.adlnet.gov/xapi/statements/more/ba419152884e57b6db8f18f8223d657fnull`

**Comment by rchadwic** [comment]
Is it possible that Query in this example is null? That would do it. I see that if you do ADL.xAPIWrapper.getStatements(null,null), this adds the "null" - which is the value of the query string. I've fixed that....

**Comment by rchadwic** [comment]
I see it. Fixed in #9ebeb2f
